cpp/tests/grouped_rolling/grouped_rolling_test.cpp

```diff
@@ -941,6 +959,23 @@ class GroupedTimeRangeRollingTest : public cudf::test::BaseFixture {
     return col.release();
   }
 
+  template <typename OutputType,
+            typename agg_op,
+            bool is_mean_of_timestamp,
+            std::enable_if_t<!is_mean_of_timestamp>* = nullptr>
```

Comment:
I've always found this explicit "is_mean" bool that gets passed around a little strange (the actual rolling code did it too). The logic for this can be inferred from the other parameters, ie
```suggestion
 template <typename OutputType,
            typename agg_op,
            std::enable_if_t<!(agg_op == MEAN && OutputType == cudf::is_timestamp<OutputType>())>* = nullptr
            >
```
Then this could be bubbled up further so that is_mean can be removed from the template params for create_reference_output

Reply:
Issue https://github.com/rapidsai/cudf/issues/5466 is created to track this.